src/main/java/teammates/ui/pagedata/InstructorCourseEnrollPageData.java

```diff
@@ -35,4 +45,29 @@ public String getEnrollStudents() {
     public void setEnrollErrorLines(Map<String, String> enrollErrorLines) {
         this.enrollErrorLines = enrollErrorLines;
     }
+
+    public void updateEnrollSuccessLines(List<StudentAttributes>[] students) {
+        for (int i = 1; i < StudentUpdateStatus.STATUS_COUNT - 2; i++) {
+            switch (StudentUpdateStatus.enumRepresentation(i)) {
+            case NEW:
+                for (StudentAttributes newStudent : students[i]) {
+                    enrollNewStudentsLines.put(newStudent.email, "Student has been added successfully.");
+                }
+                break;
+            case MODIFIED:
```

Comment:
I thought "enroll" means new students only? Otherwise it's the job of the "edit" action

Reply:
Explained more in this https://github.com/TEAMMATES/teammates/pull/9017#discussion_r208454216.